lib/compilers/carbon.ts

```diff
@@ -65,41 +67,33 @@ export class CarbonCompiler extends BaseCompiler {
         );
     }
 
-    override async afterCompilation(
-        result,
-        doExecute,
-        key,
-        executeParameters,
-        tools,
-        backendOptions,
-        filters,
-        options,
-        optOutput,
-        customBuildPath,
-    ) {
-        result = await super.afterCompilation(
-            result,
-            doExecute,
-            key,
-            executeParameters,
-            tools,
-            backendOptions,
-            filters,
-            options,
-            optOutput,
-            customBuildPath,
-        );
+    lastLine(lines?: ResultLine[]): string {
```

Comment:
There has to be a nicer way of doing this, but I couldn't placate the type system.